src/ui/common/commonActions.js

```diff
@@ -0,0 +1,63 @@
+import {action} from "mobx";
+import MainStore from "../MainStore";
+import ToastStore from "../toast/ToastStore";
+
+module.exports = {
+  tagStringToArray: tagStringToArray,
+	isDuplicateTitle: isDuplicateTitle,
+  isDuplicateTitleSkipIndex: isDuplicateTitleSkipIndex,
+	isInvalidTitle: isInvalidTitle,
+	isInvalidTags: isInvalidTags
+};
```

Comment:
Indentation + wrap the object keys in 'action'. eg. `isInvalidTags: action(isInvalidTags)` - doing this should provide [better performance as well as better debugging](https://medium.com/@mweststrate/mobx-2-2-explicit-actions-controlled-mutations-and-improved-dx-45cdc73c7c8d#.7a0rdfqga) with MobX devtools.